src/Resources/public/Admin.js

```diff
@@ -100,7 +100,7 @@ var Admin = {
         if (window.SONATA_CONFIG && window.SONATA_CONFIG.USE_ICHECK) {
             Admin.log('[core|setup_icheck] configure iCheck on', subject);
 
-            jQuery("input[type='checkbox']:not('label.btn>input'), input[type='radio']:not('label.btn>input')", subject).iCheck({
+            jQuery("input[type='checkbox']:not('label.btn>input, [data-sonata-icheck='false']'), input[type='radio']:not('label.btn>input')", subject).iCheck({
```

Comment:
I think the code is not correct i have created a fiddle with tests

problems:
* invalid css selector
* not working for radios

https://jsfiddle.net/jffjgbem/9/

the correct one should:
`input[type='checkbox']:not(label.btn>input, [data-sonata-icheck='false']), input[type='radio']:not(label.btn>input, [data-sonata-icheck='false'])`

Reply:
Hi, the single quotes syntax errors have indeed slipped through our proof reading process. This is due to the 3-level nesting of quotes, we'll patch it right now. The single quotes, in the `not` selectors, are not needed, we'll remove it at the same time.

However, we did not want to change the behaviour for the radio in the same pull request: we planned to add a new one if this one was merged.